app/src/main/java/com/example/diceroller/DiceViewAdapter.kt

```diff
@@ -1,49 +1,45 @@
 package com.example.diceroller
 
-import android.content.Context
 import android.view.LayoutInflater
 import android.view.ViewGroup
-import androidx.core.graphics.drawable.toDrawable
 import androidx.recyclerview.widget.RecyclerView
 import com.example.diceroller.databinding.ItemRolledDiceBinding
 import java.text.DateFormat
 
-class DiceViewAdapter (
-    private val items: List<Dice>
-    ): RecyclerView.Adapter<DiceViewAdapter.DiceViewHolder>() {
-
-    class DiceViewHolder(private val itemRolledDiceBinding: ItemRolledDiceBinding) :
-        RecyclerView.ViewHolder(itemRolledDiceBinding.root) {
-
-        fun bindDice(item: Dice) {
-            with(itemRolledDiceBinding) {
```

Comment:
Nice use of a scope function! Scope functions can be very powerful and reduce the need to repeat yourself.